ovs/extensions/healthcheck/alba_hc.py

```diff
@@ -321,31 +321,28 @@ def _get_all_responding_backends(result_handler):
         for alba_backend in BackendHelper.get_albabackends():
             # check if backend would be available for vpool
             try:
-                available = False
-                for preset in alba_backend.presets:
-                    if preset.get('is_available'):
-                        available = True
-                # collect ASDs connected to a backend
-                asds = []
-                for stack in alba_backend.local_stack.values():
-                    for osds in stack.values():
-                        for asd in osds['osds'].values():
-                            if alba_backend.guid != asd.get('claimed_by'):
+                # collect OSDs connected to a backend
+                osds = []
+                for local_stack in alba_backend.local_stack.itervalues():
+                    for osd_stack in local_stack.itervalues():
+                        for osd in osd_stack['osds'].itervalues():
+                            if alba_backend.guid != osd.get('claimed_by'):
                                 continue
                             else:
-                                asds.append(asd)
+                                osds.append(osd)
+
                 # create result
                 result.append({
                     'name': alba_backend.name,
                     'alba_id': alba_backend.alba_id,
-                    'is_available_for_vpool': available,
+                    'is_available_for_vpool': any(preset for preset in alba_backend.presets if preset.get('is_available') is True),
```

Comment:
Any requires an iterable. The current argument supplied is missing [ ] or ( ) (I'd opt for ())